src/CoreBundle/Resources/public/themes/standard/css/global.css

```diff
@@ -5,22 +5,68 @@ html {
 }
 
 body {
-    height: 100%;
+    overflow-x: hidden;
```

Comment:
That would be _some_ solution for surplus horizontal scrollbars.

But will be removed here.